cli/src/cmd/forge/fourbyte.rs

```diff
@@ -28,6 +28,8 @@ pub struct UploadSelectorsArgs {
 impl UploadSelectorsArgs {
     /// Builds a contract and uploads the ABI to selector database
     pub async fn run(self) -> eyre::Result<()> {
+        println!("Warning! This command is deprecated and will be removed in v1, use `forge selectors upload` instead");
```

Comment:
i think we could make this even more flashy with some yellow color, e.g: https://github.com/foundry-rs/foundry/blob/master/cli/src/cmd/forge/script/mod.rs#L750